bgpd/bgp_packet.c

```diff
@@ -697,6 +679,33 @@ void bgp_open_send(struct peer_connection *connection)
 	/* Set BGP packet length. */
 	bgp_packet_set_size(s);
 
+	return s;
+}
+
+/*
+ * Creates a BGP Open packet and appends it to the peer's output queue.
+ * Sets capabilities as necessary.
+ */
+void bgp_open_send(struct peer_connection *connection)
+{
+	struct stream *s;
+	uint16_t send_holdtime;
+	as_t local_as;
+	struct peer *peer = connection->peer;
+
+	if (CHECK_FLAG(peer->flags, PEER_FLAG_TIMER))
+		send_holdtime = peer->holdtime;
+	else
+		send_holdtime = peer->bgp->default_holdtime;
```

Comment:
i agree but i also need them for the debug message:
```c
if (bgp_debug_neighbor_events(peer))
    zlog_debug(
	    "%s sending OPEN, version %d, my as %u, holdtime %d, id %pI4",
	    peer->host, BGP_VERSION_4, local_as, send_holdtime,
	    &peer->local_id);
```

 if i move the message in bgp_open_make() it will log when i generated bgp opens in bmp too.
i can instead turn 
```c
if (CHECK_FLAG(peer->flags, PEER_FLAG_TIMER))
	send_holdtime = peer->holdtime;
else
	send_holdtime = peer->bgp->default_holdtime;
```
into a bgp_peer_get_send_holdtime()